src/App.tsx

```diff
@@ -277,7 +279,7 @@ export default function App() {
           <ChatUIProvider
             user={userPushSDKInstance}
             theme={darkMode && darkChatTheme}
-            debug={true}
+            debug={pushsdkDebug}
```

Comment:
```suggestion
            debug={true}
```

It's fine this way as we are not using it anywhere else.